browser/src/control/jsdialog/Widget.MultilineEdit.ts

```diff
@@ -0,0 +1,149 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * JSDialog.MultilineEdit - text field with multiple lines and scrollbar
+ *
+ * Example JSON:
+ * {
+ *     id: 'id',
+ *     type: 'multilineedit',
+ *     text: 'text content\nsecond line',
+ *     html: '<a href="hyperlink">hyperlink</a>' // only if contenteditable is true
+ *     cursor: true,
+ *     contenteditable: false
+ *     enabled: false
+ * }
```

Comment:
thank you very much fro help!
In typescript we try to take advantage of types/interfaces to check correctness.
So instead of example JSON comment which can be out of date, we add new interface definition to:
See an example: https://github.com/CollaboraOnline/online/blob/master/browser/src/control/jsdialog/Definitions.Types.ts#L274

So please remove Example JSON comment and move new interface to mentioned file (`MultilineEditData`)